Moli/API/UserAPI/UserSubpath.swift

```diff
@@ -139,4 +139,14 @@ extension UserAPI {
             expectingReturnType: IsBlockedResponse.self
         )
     }
+
+    static public func reportUser(userId: String, reason: String, description: String) async throws {
+        let body = ReportRequest(reportedUserId: userId, reason: reason, description: description)
```

Comment:
:art: Vi brukar köra på en tum-regel där om man ska skicka fler än 1 parameter, som vi gör här tex där vi skickar in tre, så kan man istället ha en parameter som heter `params` som typningen är dess struct, i det här fallet `ReportRequest`. Då kan man jobba enklare med klass instansen utanför koden

Så att man istället kan göra så att:
```suggestion
    static public func reportUser(params: ReportRequest) async throws {
```